azure-quantum/tests.live/Run.ps1

```diff
@@ -76,6 +76,10 @@ $logs = Join-Path $env:BUILD_ARTIFACTSTAGINGDIRECTORY "logs" "qdk-python.txt"
 
 # Copy unit tests without recordings and run Pytest
 Copy-Item -Path (Join-Path $PackageDir "tests" "unit" "*.py") -Destination $PSScriptRoot
+if ($PackageDir -contains "azure-quantum") {
```

Comment:
Apparently, `-contains` doesn't work for string comparisons in Powershell (see: https://stackoverflow.com/questions/18877580/powershell-and-the-contains-operator)

I was very confused because the tests failed again, but I think this is the culprit (new test run at: https://ms-quantum.visualstudio.com/Quantum%20Program/_build/results?buildId=262862&view=logs&j=12427bb5-18dc-5861-5010-3c822612ce46&t=1697fa8b-04ed-518c-861e-2be46dc0a342)

Reply:
Thanks for the pointer. I changed `contains` to `match`.

Reply:
Tests are passing now.